src/core/mac/sub_mac.cpp

```diff
@@ -253,47 +252,6 @@ Error SubMac::Receive(uint8_t aChannel)
     return error;
 }
 
-#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
-Error SubMac::CslSample(uint8_t aPanChannel)
```

Comment:
I think `CslSample()` in the PR is now moved after the new `SubMac::UpdateCsl()`. Maybe better to move it here to help reduce the `git diff` (we can re-arrange the method in a later PR).

Reply:
Makes sense. Done.